packages/api/src/consts.types.ts

```diff
@@ -0,0 +1,95 @@
+// Auto-generated via `yarn build:interfaces`, do not edit
+/* eslint-disable @typescript-eslint/no-empty-interface */
+
+import { Bytes, u32, u64 } from '@polkadot/types';
+import { Balance, BalanceOf, BlockNumber, Moment, Permill } from '@polkadot/types/interfaces/runtime';
+import { Gas } from '@polkadot/types/interfaces/contracts';
+import { VoteIndex } from '@polkadot/types/interfaces/elections';
+import { SessionIndex } from '@polkadot/types/interfaces/session';
+import { EraIndex } from '@polkadot/types/interfaces/staking';
+import { Codec } from '@polkadot/types/types';
+
+declare module '@polkadot/metadata/Decorated/types' {
+  export interface Constants {
+    babe: {
+      [index: string]: Codec;
```

Comment:
The reason why this is not a ConstantCodec is that with ConstantCodec, users cannot do `api.const.a.b as Balance` without a TS error "Balance has not enough overlapping with ConstantCodec"

Reply:
That is fine. Really don't see any issue with it as it stands. (It actually simplifies things somewhat, since we use Codec everywhere so it it not a new type that comes in)